contrib/cirrus/setup_environment.sh

```diff
@@ -79,6 +79,10 @@ then
     if run_rootless
     then
         setup_rootless
+        make install.catatonit
+        go get github.com/onsi/ginkgo/ginkgo
+        go get github.com/onsi/gomega/...
+        dnf -y update runc
```

Comment:
These things should probably all be done while building VM images.  No reason to eat that extra time for every PR.  I can help sort that out later though.